app/src/core/constants/theme.ts

```diff
@@ -1,52 +1,182 @@
-import { ThemeOptions } from '@mui/material';
```

Comment:
Along the same lines as above I'm not sure we want to purge/swap the colors for what PLUT has here. I don't hate/have an opinion on taking the overrides for any of the typography